### PR TITLE
improve development setup instructions to leave no questions open

### PR DESCRIPTION
**Pull request by jancborchardt**
@xmartin as you know I’m really a noob, so the readme wasn’t easy enough in how to actually run a web server without looking it up. I thought it would be cool if we list an example command which can simply be copied. It should work on most Linuxes and macOS, right?

Even comes with a simple step-divided instruction. :heart:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,12 +8,11 @@ The no-login variant is currently in alpha and is supposed to replace the [remot
 
 The UI is built using ReactJS and Redux.
 
-## Setup
+## Development setup
```
Comment (xMartin) [inline README.md]: We have "for development" and "for production" sections here so I'd leave "##Setup".

File: README.md [hunk 2]
```diff
@@ -8,12 +8,11 @@ The no-login variant is currently in alpha and is supposed to replace the [remot
...
 
-`npm install` and copy `js/config-sample.js` to `js/config.js`.
-
-Just serve the files in this directory with any static web server.
-
-For development, open `index.dev.html` in the browser.
+1. Run `npm install` in this folder
+2. Copy `js/config-sample.js` to `js/config.js`
+3. Start any static web server in this folder, for example with `python -m SimpleHTTPServer 8000`
```
Comment (xMartin) [inline README.md]: I like the addition of the example.

**Comment by xMartin** [comment]
As an alternative we could also easily add a web server as a nodejs dependency in a way that you just have to run `npm run serve` or something.

**Comment by jancborchardt** [comment]
@xMartin I reset the header to »Setup«. :)

**Pull request merged**